src/core/mac/link_raw.cpp

```diff
@@ -70,7 +72,22 @@ LinkRaw::LinkRaw(Instance &aInstance)
     , mEnergyScanDoneCallback(NULL)
 {
     // Query the capabilities to check asserts
-    (void)GetCaps();
+    mRadioCaps     = otPlatRadioGetCaps(&GetInstance());
```

Comment:
Suggestion: How about adding `mRadioCaps` and `mTransmitFrame` to the constructor initializer list?